src/rtc/h5_prep.py

```diff
@@ -265,6 +268,7 @@ def get_metadata_dict(product_id: str,
         #    [mission_id, 'Mission identifier'],
         'identification/platform':
             ['platform', platform_id, 'Platform name'],
+        # https://sentinel.esa.int/web/sentinel/missions/sentinel-1/satellite-description
```

Comment:
So it looks like the S-1 sensor is indeed called `CSAR`?

Reply:
Yes, according to the ESA webpage. But I think it's okay to use `platform_id` here as we discussed.

Reply:
Sounds good. I'll update the reference to the PDF document you posted above.